.github/ISSUE_TEMPLATE/BUG.yml

```diff
@@ -45,16 +45,16 @@ body:
   - type: input
     id: obsndi-version
```

Comment:
Should this id be updated as well ?

Reply:
I see I missed a few renames.
I wasn't wearing my glasses when I was doing it initially and my regex was incorrectly `obs[-]ndi` instead of what it should have been as `obs(-?)ndi`.
Looks like I've got at least one more push to make!

Reply:
Resolving this; will be in my next push.